cli/commands/genesis/payload.go

```diff
@@ -107,7 +107,7 @@ func AddExecutionPayload(chainSpec ChainSpec, elGenesisPath string, config *cmtc
 		return errors.Wrap(err, "failed to unmarshal beacon state")
 	}
 	// Inject the execution payload.
-	eph, err := executableDataToExecutionPayloadHeader(
+	eph, err := ExecutableDataToExecutionPayloadHeader(
```

Comment:
I don't think we should change the accessibility of existing beacon-kit functions. Let's create a new one that calls this internally